src/core/core.tooltip.js

```diff
@@ -87,6 +87,9 @@ module.exports = function(Chart) {
 					backgroundColor: view.backgroundColor
 				};
 			},
+			textLabelColor: function(tooltipItem, chart) {
+				return chart.tooltip._options.bodyFontColor;
```

Comment:
it might be an idea here to pass the `tooltip` object as a parameter so that we're not reliant on it being a property of the chart. I know some users programmatically add extra tooltips and this would break that